packages/entities/entities-plugins/src/plugins-endpoints.ts

```diff
@@ -9,14 +9,26 @@ export default {
       forEntity: '/{workspace}/{entityType}/{entityId}/plugins',
     },
   },
+  select: {
+    konnect: {
+      availablePlugins: '/api/runtime_groups/{controlPlaneId}/v1/available-plugins',
+    },
+    kongManager: {
+      availablePlugins: '/{workspace}/kong',
```

Comment:
Non-blocking: just realized if we want to adopt `PluginSelect` in the [open-source Kong Manager](https://github.com/Kong/kong-manager), we'll need to update this block because Kong Gateway CE has a different endpoint to return available plugins. But don't worry, the KM team can handle this when we start to uptake `PluginSelect`.